interface/forms/LBF/new.php

```diff
@@ -1553,7 +1553,7 @@ function warehouse_changed(sel) {
 // New form and this patient has a portal login and we have not loaded portal data.
 // Check if there is portal data pending for this patient and form type.
 if (!$alertmsg && !$formid && $GLOBALS['gbl_portal_cms_enable'] && $cmsportal_login && !$portalid) {
-    $portalres = cms_portal_call(array('action' => 'checkptform', 'form' => $formname, 'patient' => $cmsportal_login));
+    $portalres = cms_portal_call(array('action' => 'checkptform', 'form' => attr($formname), 'patient' => $cmsportal_login));
```

Comment:
Take a look at that function and see if you need to escape this (hint: you don't :) ).

Reply:
you are right :)